src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js

```diff
@@ -6,10 +6,18 @@ Cypress.Commands.add('checkSearch', () => {
   cy.axeCheck();
 
   // Search
+  cy.get('#street-city-state-zip', { timeout: 10000 })
+    .should('exist')
+    .should('not.be.disabled')
+    .clear({ waitForAnimations: true, force: true });
+
+  // This forEach loop is a workaround to a typing bug in Cypress.
```

Comment:
Can we please add the URL of the bug report to the comment? Thanks!

Reply:
Just added it.